lxd/device/config/devices.go

```diff
@@ -3,6 +3,7 @@ package config
 import (
 	"fmt"
 	"sort"
+	"strings"
```

Comment:
You're adding an import here without ever using it, this should cause a compilation failure.